pkg/network/ovn_kubernetes_test.go

```diff
@@ -167,8 +178,12 @@ func TestOVNKubernetesIsSafe(t *testing.T) {
 	// change the mtu
 	mtu := uint32(70000)
 	next.DefaultNetwork.OVNKubernetesConfig.MTU = &mtu
-	errs = isOVNKubernetesChangeSafe(prev, next)
-	g.Expect(errs).To(HaveLen(1))
 
-	g.Expect(errs[0]).To(MatchError("cannot change ovn-kubernetes configuration"))
+	// change the geneve port
+	geneve := uint32(70001)
```

Comment:
you should use a valid port value here

Reply:
The test is for an invalid port.

Reply:
The test for an invalid port is earlier; this line is in the test that changes to the config aren't allowed, but you used the same invalid port. It's better to test this with a valid port so that we're confirming that you can't change it even if you try to change it to a different valid port.

Reply:
Changed